pkg/controller/machine.go

```diff
@@ -708,6 +731,11 @@ func (c *controller) updateMachineConditions(machine *v1alpha1.Machine, conditio
 		}
 		glog.V(2).Infof(msg)
 
+		err = c.deleteBootstrapToken(clone.Name)
```

Comment:
If this is to be done only while machine creation occurs, we could move it one block up to here - https://github.com/gardener/machine-controller-manager/blob/b2dfbebabc8c57f4dee0114fa37bd921e7451812/pkg/controller/machine.go#L722-L726

Reply:
10x . I have moved it as you told me.